paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentOptionFactory.kt

```diff
@@ -18,6 +18,12 @@ internal class PaymentOptionFactory(
                     label = resources.getString(R.string.google_pay)
                 )
             }
+            PaymentSelection.Link -> {
+                PaymentOption(
+                    drawableResourceId = R.drawable.stripe_ic_paymentsheet_link,
+                    label = "Link"
```

Comment:
Does this need to be translated?

Reply:
No because it's the brand name, but I'll move it into a resources file in the next PR.